MovieQuiz/Services/MoviesLoader.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  MoviesLoader.swift
+//  MovieQuiz
+//
+//  Created by Кирилл Брызгунов on 25.11.2022.
+//
+
+import Foundation
+
+protocol MoviesLoading {
+    func loadMovies(headler: @escaping (Result<MostPopularMovies, Error>) -> Void)
+}
+
+
+struct MoviesLoader: MoviesLoading {
+    // MARK: - NetworkClient
+    private let networkClient = NetworkClient()
+    
+    // MARK: - URL
+    private var mostPopularMoviesUrl: URL {
+        guard let url = URL(string: "https://imdb-api.com/en/API/Top250Movies/k_adn67lp5") else {
+            preconditionFailure("Unable to construct mostPopularMoviesUrl")
+        }
+        return url
+    }
+    
+    func loadMovies(headler: @escaping (Result<MostPopularMovies, Error>) -> Void) {
```

Comment:
```suggestion
    func loadMovies(handler: @escaping (Result<MostPopularMovies, Error>) -> Void) {
```

Можно включить[ проверку правописания, чтобы не оставлять опечаток](https://sarunw.com/posts/spell-checking-in-xcode/)